src/main/webapp/scripts/my-profile.js

```diff
@@ -39,6 +39,9 @@ function loadDetails() {
     } else {
       const user = await response.json();
       // Populate user details section
+      if ( !user.hasOwnProperty("profilePictureUrl") ) {
```

Comment:
```suggestion
      if (!user.hasOwnProperty("profilePictureUrl") ) {
```

Same below please :)